public/app/plugins/datasource/elasticsearch/datasource.ts

```diff
@@ -553,10 +546,6 @@ export class ElasticDatasource
   }
 
   getSupplementaryQuery(options: SupplementaryQueryOptions, query: ElasticsearchQuery): ElasticsearchQuery | undefined {
-    if (!this.getSupportedSupplementaryQueryTypes().includes(options.type)) {
-      return undefined;
-    }
-
```

Comment:
Why is this removed?

Reply:
There's no need to add that, when in both cases the switch is already doing that work:

https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/elasticsearch/datasource.ts#L626

https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/loki/datasource.ts#L249